docs/source/model_design/programming_vivarium/running_simulations/index.rst

```diff
@@ -31,9 +31,207 @@ Running Simulations
    :local:
    :depth: 1
 
+Once you have your model completed, you'll need to actually run a simulation. 
+There is documentation on an `engineering page <https://vivarium.readthedocs.io/en/latest/tutorials/running_a_simulation/index.html>`_ and `on the Hub <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=SSE&title=Running+a+parallel+simulation>`_ about 
+running the simulation, but here 
+will make a how-to guide for running simulations for a research team 
+audience. The below sections are meant to guide someone through running 
+a simulation and the various sub-steps. 
+
+Data Inputs and Model Spec Files
+--------------------------------
+
+Prior to running a simulation, check all the input data and 
+parameters match the run you want. 
+
+Data Inputs
++++++++++++
+
+The data inputs for the simulation should all be in the artifact. Check 
+out our page on :ref:`building the artifact <artifact_building_rt>` if 
+you need more information on this. 
+
+The model_spec File
++++++++++++++++++++
+
+The :code:`model_spec.yaml` file contains the majority of the parameters 
+for a given model run. This includes things like what 
+components to include, the population size, what draw and artifact to use, 
+and stratifications for observers. The research team now owns this file. 
+
+Engineering notes can be found on this `model specs file page <https://vivarium.readthedocs.io/en/latest/concepts/model_specification/index.html>`_.
+
+At the top, there is a section titled :code:`components` which lists the 
+components in the model. You might see subheadings, which refer to the 
+names of specific packages, and within those packages are the components 
+themselves. Most of the components will be in the engineering repository, 
+however some are often from :code:`vivarium_public_health` as they are used in 
+multiple simulations. 
+
+Generally, research will not be responsible for writing or maintaining 
+components, though this might change in the future. You'll mainly use this 
+section to turn on or off components. 
+
+Below is the :code:`configuration` section, which contains the specifications 
+for a single model run, including the draw number, artifact path, population, 
+scenario, time period, and any stratifications. This should largely be 
+familiar as RT members edit this frequently for interactive sim runs. 
```

Comment:
Link to docs on interactive sims, in case a reader is not frequently editing for this yet?